nailgun/entities.py

```diff
@@ -6825,8 +6823,13 @@ def create_payload(self):
         For more information, see `Bugzilla #1151220
         <https://bugzilla.redhat.com/show_bug.cgi?id=1151220>`_.
 
+        In addition, rename the ``from_`` field to ``from``.
+
         """
-        return {u'subnet': super(Subnet, self).create_payload()}
+        payload = super(Subnet, self).create_payload()
+        if 'from_' in payload:
+            payload['from'] = payload.pop('from_')
+        return {u'subnet': payload}
```

Comment:
Great, thanks. Now to the coverage - you should create test in `CreatePayloadTestCase` (similar to `test_discovery_rule`), `ReadTestCase` (same as existing `test_discovery_rule` there) and `UpdatePayloadTestCase` (same as `test_discovery_rule_search`).

Reply:
@abalakh, done, could you review them? thanks